crates/glaredb/src/rpc_proxy.rs

```diff
@@ -47,7 +47,7 @@ impl RpcProxy {
             let cert = std::fs::read_to_string(CERT_PATH)?;
             let key = std::fs::read_to_string(CERT_KEY_PATH)?;
 
-            let identity = Identity::from_pem(cert, key);
+            let identity = Identity::from_pem(&cert, &key);
```

Comment:
I think this was an "oops" on my part, see:

https://docs.rs/tonic/latest/tonic/transport/index.html#server

Reply:
This won't fix the issue.

The function takes `AsRef<[u8]>` which means `cert` and `&cert` are both valid (since they implement `AsRef`).

https://docs.rs/tonic/latest/tonic/transport/struct.Identity.html#method.from_pem

Reply:
Likely wildcard then, just wanted to cover my bases. I'll close.